tensorbay/utility/common.py

```diff
@@ -3,14 +3,7 @@
 # Copyright 2021 Graviti. Licensed under MIT License.
 #
 
-"""Common_loads method, EqMixin class.
-
-:meth:`common_loads` is a common method for loading an object from a dict or a list of dict.
-
-:class:`EqMixin` is a mixin class to support __eq__() method,
-which compares all the instance variables.
-
-"""
+"""Basic concepts of related deprecated functions."""
```

Comment:
what is the difference between `Basic concepts of related deprecated functions.` and `Basic concepts of deprecated related functions.`?